controllers/api.go

```diff
@@ -0,0 +1,78 @@
+package controllers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/FideTech/yaus/core"
+	"github.com/FideTech/yaus/models"
+	"github.com/FideTech/yaus/storage"
+)
+
+func ApiHandler(w http.ResponseWriter, r *http.Request) {
+	path := strings.Replace(r.URL.Path, "/api/v1", "", 1)
+
+	if strings.HasPrefix(path, "/short-links") {
+		shortLinksHandler(w, r, path)
+		return
+	}
+}
+
+func shortLinksHandler(w http.ResponseWriter, r *http.Request, path string) {
+	subPath := strings.Replace(path, "/short-links", "", 1)
```

Comment:
One limitation of not checking but just stripping is basically if I post to /api at all no matter if I goto /api/short-links or not I’ll be able to talk to this api right?

Reply:
I check for that above in the function which calls this one

Reply:
Ah missed that